config/code_schools.yml

```diff
@@ -438,6 +438,13 @@
   va_accepted: false
   full_time: true
   hardware_included: false
-  
-    
-  
+
+- name: Codeup
+  url: http://codeup.com/
+  address1: 600 Navarro St., #350
```

Comment:
If this is something that I can and should test locally, I'll try it on a different computer. Is it not showing up for you?